astroplan/core.py

```diff
@@ -161,6 +163,46 @@ def altaz(self, time, target=None, obswl=None):
                 coordinate = target
             return coordinate.transform_to(altaz_frame)
 
+    def parallactic_angle(self, time, target):
+        '''
+        Calculate the parallactic angle.
```

Comment:
@bmorris3 – Please add the reference explaining what parallactic angle is to the docstring.